terraform/helm/aptos-node/files/test-data/secure-data.json

```diff
@@ -0,0 +1 @@
+{"safety_data":{"data":"GetResponse","last_update":1651079268,"value":{"epoch":0,"last_voted_round":0,"preferred_round":0,"one_chain_round":0,"last_vote":null}}, "genesis-waypoint":{"data":"GetResponse","last_update":1651079196,"value":"0:9cd39dc0d461051bbe29d613ffcd698efef8c34f21615b74df9e3d146c07f6ff"},"waypoint":{"data":"GetResponse","last_update":1651079196,"value":"0:9cd39dc0d461051bbe29d613ffcd698efef8c34f21615b74df9e3d146c07f6ff"}}
```

Comment:
@sherry-x, this looks reasonable to me. What's the error you're seeing when deploying? Also, is there an easy way for me test this locally if you can't manage to debug it? Want to make sure you're unblocked. 🤔

Reply:
Aah, I think epoch should be set to `1`? Maybe try that? There's places in the code where safety data is initialized and the epoch is set to 1 😄 https://github.com/aptos-labs/aptos-core/blob/c68e56522a6f65f238fa7056d1a632b7dc2416c4/consensus/safety-rules/src/persistent_safety_storage.rs#L55